### PR TITLE
feat: hide org admin tabs when youre a member

### PR DESCRIPTION
**Pull request by sean-brydon**
### How to test?

Please check if the settings tabs are displayed accurately based on the user role (regular user, admin, organization admin/owner). Make sure the tabs are being rendered correctly depending up on the user permissions in different user roles.

Fixes: CAL-3474
Fixes: #14666 

MEMBER

ADMIN

### CODE REVIEW

**Comment by sean-brydon** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/calcom/cal.com/14667?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#14667** <a href="https://app.graphite.dev/github/pr/calcom/cal.com/14667?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#14652** <a href="https://app.graphite.dev/github/pr/calcom/cal.com/14652?utm_source=stack-comment-icon" target="_blank">Graphite</a>: 1 other dependent PR ([#14653](https://github.com/calcom/cal.com/pull/14653) <a href="https://app.graphite.dev/github/pr/calcom/cal.com/14653?utm_source=stack-comment-icon" target="_blank">Graphite</a>)
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @sean-brydon and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: packages/features/settings/layouts/SettingsLayout.tsx [hunk 1]
```diff
@@ -135,41 +135,59 @@ tabs.find((tab) => {
 // The following keys are assigned to admin only
 const adminRequiredKeys = ["admin"];
 const organizationRequiredKeys = ["organization"];
+const organizationAdminKeys = ["privacy", "appearance", "billing", "OAuth Clients", "SSO", "directory_sync"];
```
Comment (Udit-takkar) [inline packages/features/settings/layouts/SettingsLayout.tsx:138]: Screenshot 2024-04-19 at 3 40 23 PM

even 'members' tab should be hidden

**Review by Udit-takkar — COMMENTED** [review]
Everything else works fine. let me know if we want to hide 'members' tab or not

File: packages/features/settings/layouts/SettingsLayout.tsx (hunk 1 above)
Comment (sean-brydon) [reply packages/features/settings/layouts/SettingsLayout.tsx:138]: This should only be hidden when an org is private - Will do this in a follow up PR as it requries fetching this info from somwhere and im trying to get around adding another fetch in here for a loading state

**Review by Udit-takkar — APPROVED** [review]

**Pull request merged**